tests/integration/targets/activation/tasks/main.yml

```diff
@@ -133,6 +134,24 @@
         organization_name: Default
       register: _result
 
+    - name: Create a new credential for DE
+      ansible.eda.credential:
```

Comment:
Can you make sure that this credential is also cleaned up? https://github.com/ansible/event-driven-ansible/pull/405/files#diff-8114fbc7fb8c6410b76a6d730ecc9747780e0058096ea4bf28471d65a8665c1dR583 probably needs to be updated.

Reply:
updated. Thanks!